### PR TITLE
Adding entity names to host inventory queries for providers and actors

### PR DESCRIPTION
**Pull request by autodidaddict**

### CODE REVIEW

File: crates/wasmcloud-host/src/host_controller/hc_actor.rs [hunk 1]
```diff
@@ -548,12 +548,26 @@ impl Handler<QueryHostInventory> for HostController {
         Box::pin(
             async move {
                 let image_refs = lc.collect_oci_references().await;
+                let names: HashMap<String, String> = {
+                    // This is semi-wasteful (incurs 1 extra hashmap lookup per entity),
+                    // but I can't seem to find a good way to call the async send while inside the other loop
+                    // because async !@$%@#$)%@@#$ 😡😡😡😡
```
Comment (brooksmtownsend) [inline crates/wasmcloud-host/src/host_controller/hc_actor.rs:554]: I tried too and I can concur, async definitely !@$%@#$)%@@#$ 😡😡😡😡

**Review by brooksmtownsend — CHANGES_REQUESTED** [review]
Only requested change pertains to moving the `widl` here re: https://github.com/wasmCloud/actor-interfaces/pull/76#issuecomment-818926447

**Review by brooksmtownsend — DISMISSED** [review]

**Review by brooksmtownsend — APPROVED** [review]
if tests pass I'm happy 👍🏻

**Comment by autodidaddict** [comment]
> if tests pass I'm happy 👍🏻

I test in prod :shipit:

**Pull request merged**